main.yml

```
@@ -766,6 +764,66 @@ service-registry:
         title: Documentation
         navigate: https://access.redhat.com/documentation/en-us/red_hat_openshift_service_registry
 
+
+## App Services (RHOAS) APIs
+app-services-apis:
+  title: Application Services
+  api:
+    subItems:
+      kafkamgmt:
+        title: "OpenShift Streams for Apache Kafka Management API"
+        api:
+          github:
+            owner: redhat-developer
+            repo: app-services-sdk-core
+            path: kas-fleet-manager.yaml?ref=doc-portal
```

Extra `api` and extra spacing
```suggestion
        github:
          owner: redhat-developer
          repo: app-services-sdk-core
          path: kas-fleet-manager.yaml?ref=doc-portal
```